Dockerfile

```diff
@@ -1,5 +1,38 @@
 FROM python:3.9
 
+WORKDIR /data
+
+# Install tools necessary used to install samtools and htslib so we can configure fasta files for genomic assembly.
+RUN apt-get update && apt-get install -y \
```

Comment:
If we don't need these tools when running, it might be worth doing this as a [multi-stage build](https://docs.docker.com/build/building/multi-stage/), just so all that build-essential type stuff isn't left behind in the final container.  It's a pretty small change to what you've already got, just add in: 
```FROM python:3.9 AS downloader```
up the top and then
```FROM python:3.9
COPY --from=downloader /data /data```
before `WORKDIR /code`.

Reply:
Oh cool, didn't even know about that- thanks for the tip! I'll try it out tomorrow AM.